openslides_backend/action/actions/user/set_present.py

```diff
@@ -89,3 +93,23 @@ def check_permissions(self, instance: Dict[str, Any]) -> None:
         ):
             return
         raise PermissionDenied("You are not allowed to set present.")
+
+    def get_history_information(self) -> Optional[List[str]]:
+        # use present_values to get present_str
+        present_str = "(not) "
```

Comment:
String for when multiple presence values are given: "Changed presence in [...]"

Reply:
Done.